jobs/redis/templates/config/redis.conf.erb

```diff
@@ -105,9 +105,19 @@ databases 16
 #
 #   save ""
 
-save 900 1
-save 300 10
-save 60 10000
+<% if_p('redis_save_intervals') do
+    redis_save_intervals = p('redis_save_intervals')
+
+    if redis_save_intervals.length > 0
+        redis_save_intervals.each do | command |
```

Comment:
Personally I like this. Bosh added -%> over the last year or so. So if we add this then we strict the release to relative new bosh directors. I'm personally fine with this. If someone complains - and cannot upgrade their bosh - then we can remove the -%> brackets.